package.json

```diff
@@ -14,30 +20,33 @@
     "@angular/platform-browser-dynamic": "^8.0.0",
     "@types/jasmine": "^3.3.13",
     "awesome-typescript-loader": "^5.2.1",
-    "copyfiles": "^2.1.0",
```

Comment:
also this as well

Reply:
I removed it, because the copy is not neede anymore, since the files all end up in the `dist/ngx-buildplus` folder... is it needed for anything I missed?